presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/SchemaDao.java

```diff
@@ -130,4 +131,11 @@
             "  FOREIGN KEY (node_id) REFERENCES nodes (node_id)\n" +
             ")")
     void createTableBuckets();
+
+    @SqlUpdate("CREATE TABLE IF NOT EXISTS shard_organizer_jobs (\n" +
+            "  node_identifier VARCHAR(255) NOT NULL,\n" +
+            "  last_run_time DATETIME NOT NULL,\n" +
+            "  PRIMARY KEY (node_identifier)\n" +
+            ")")
+    void createShardOrganizerJobs();
```

Comment:
Feel free to add a commit before this one to remove `Table` from the method names